cmd/conf/config.go

```diff
@@ -14,23 +17,23 @@ const PASSWORD_NOT_SET = "PASSWORD_NOT_SET"
 type ConfConfig struct {
 	Dump      bool     `koanf:"dump"`
 	EnvPrefix string   `koanf:"env-prefix"`
-	File      string   `koanf:"file"`
+	File      []string `koanf:"file"`
 	S3        S3Config `koanf:"s3"`
 	String    string   `koanf:"string"`
 }
 
 func ConfConfigAddOptions(prefix string, f *flag.FlagSet) {
 	f.Bool(prefix+".dump", ConfConfigDefault.Dump, "print out currently active configuration file")
 	f.String(prefix+".env-prefix", ConfConfigDefault.EnvPrefix, "environment variables with given prefix will be loaded as configuration values")
-	f.String(prefix+".file", ConfConfigDefault.File, "name of configuration file")
+	f.StringSlice(prefix+".file", ConfConfigDefault.File, "name of configuration file")
 	S3ConfigAddOptions(prefix+".s3", f)
 	f.String(prefix+".string", ConfConfigDefault.String, "configuration as JSON string")
 }
 
 var ConfConfigDefault = ConfConfig{
 	Dump:      false,
 	EnvPrefix: "",
-	File:      "",
+	File:      []string{""},
```

Comment:
Should this default to nil instead or does that not work?

Reply:
:+1: